schema/rdl_schema_0.1.json

```
@@ -1025,6 +788,104 @@
 				"Judgement",
 				"Hybrid"
 			]
+		},
+		"Spatial": {
+			"title": "Spatial coverage",
+			"type": "object",
+			"description": "Information about the spatial coverage of the dataset.",
+			"properties": {
+				"countries": {
+					"title": "Countries",
+					"type": "array",
+					"description": "The countries covered by the geographical area, from the open country codelist (ISO 3166-1).",
+					"items": {
+						"type": "string"
+					},
+					"minItems": 1
+				},
+				"gazetteerEntries": {
+					"title": "Gazetteer entries",
+					"type": "array",
+					"description": "Entries from geographical indices or directories describing the geographical area. This field should be used to describe subnational coverage. Use of is recommended.",
```

Per https://github.com/GFDRR/rdl-standard/issues/52#issuecomment-1576859908

```suggestion
					"description": "Entries from geographical indices or directories describing the geographical area. This field should be used to describe subnational coverage. Use of ISO 3166-2 is recommended.",
```